src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -218,7 +203,9 @@ const NavHeader = (props) => {
       {hasSearch && (
         <View
           style={{
-            width: '100%'
+            width: '100%',
+            borderBottomWidth: 1,
+            borderBottomColor: COLORS.gray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a border to the search bar improves its visibility. Good UI improvement.



<!-- This is an auto-generated comment by OSS CodeRabbit -->